tiny_obj_loader.h

```diff
@@ -1892,15 +1928,25 @@ bool LoadObj(attrib_t *attrib, std::vector<shape_t> *shapes,
     // object name
     if (token[0] == 'o' && IS_SPACE((token[1]))) {
```

Comment:
Object name. I have not tested this part with an example. I suppose your unit tests will have an example to cover this.